service/route.go

```diff
@@ -134,5 +135,12 @@ func (r *Route) serveHTTP(ctx *httpserve.Context) (res httpserve.Response) {
 		return httpserve.NewTextResponse(400, []byte(err.Error()))
 	}
 
+	switch strings.ToLower(ctx.Request.Method) {
```

Comment:
Can you check if type is not `AdoptResponse`? Also can you move this logic into a helper func?

Reply:
What is AdoptResponse? I can't find any reference to it in the code.

Can you give me an example of how to check that?

Thanks!